src/main/java/dk/kb/discover/SolrService.java

```diff
@@ -349,6 +350,109 @@ private void addParamIfAvailable(UriBuilder builder, String key, Object value) {
         }
     }
 
+    /**
+     * Remove filters with {@code prefix} from the {@code params.fq} entries in {@code solrResponse}.
+     * Intended use is to remove internal licensing information from responses returned to external caller.
+     * <p>
+     * Note: This uses regexp-based search/replace to adjust JSON & XML.
+     *       This is error prone and should generally be avoided.
+     *       It is used here as the service is expected to under heavy load so performance is a high priority.
+     * @param solrResponse a Solr search response in {@code wt} format.
+     * @param prefix the prefix identifying the filter to remove, e.g. {@code {!cache=true}}.
+     * @param wt the delivery format ({@code json}, {@code xml}, {@code csv}). null means {@code json}.
+     * @return the response with the prefixed filter removed.
+     * @throws IllegalArgumentException if the {@code wt} was unknown or the {@code solrResponse} did not contain
+     *                                  a prefixed filter.
+     */
+    public static String removePrefixedFilters(String solrResponse, String prefix, String wt) {
+        switch (wt == null ? "json" : wt) {
+            case "json": return removePrefixedFiltersJSON(solrResponse, prefix);
+            case "xml": return removePrefixedFilterXML(solrResponse, prefix);
+            case "csv": return solrResponse;
```

Comment:
Yep. `csv` is only legends at the top + data.

...Thinking about it it might be an idea to have a comment line at the top stating request parameters, the same way that Labs-API does it. Not a high priority.